### PR TITLE
Add a new Allinea easyconfig for the latest 6.1.1 version.

### PR DESCRIPTION
**Pull request by akesandgren**
This also uses the new allinea easyblock to handle templates and
system.config if requested.

Depends on easyblock PR# 1025 to use the added features.

### CODE REVIEW

**Comment by akesandgren** [comment]
I guess the tests needs a rerun now that easyblock PR #1025 is merged

**Comment by boegel** [comment]
@akesandgren different issue:

```
AssertionError: Filename '/home/travis/build/hpcugent/easybuild-easyconfigs/easybuild/easyconfigs/a/Allinea/Allinea-6.1.1-Ubuntu-14.04-x86_64.eb' of parsed easyconfig matches expected filename 'Allinea-6.1.1.eb'
```

File: easybuild/easyconfigs/a/Allinea/Allinea-6.1.1-Ubuntu-14.04-x86_64.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+#    'kebnekaise.qtf',
+#    'kebnekaise-gpu.qtf',
+#]
+
+# Example of sysconfig usage
+#sysconfig = 'system.config.hpc2n'
+
+# license file
+import os
+license_file = os.path.join(os.getenv('HOME'), "licenses", "allinea", "license.lic")
```
Comment (boegel) [inline easybuild/easyconfigs/a/Allinea/Allinea-6.1.1-Ubuntu-14.04-x86_64.eb]: @akesandgren please use

```python
license_file = HOME + 'licenses/allinea/license.lic'
```

using `import` in easyconfig files is something we should try and avoid..

do you mind also doing this in the existing easyconfig for Allinea?

**Comment by akesandgren** [comment]
Yeah, can fix that in the older ones too,  in a new PR or add it to this one?

**Comment by boegel** [comment]
@akesandgren here is fine, trivial fix & easy to test

File: easybuild/easyconfigs/a/Allinea/Allinea-4.1-32834-Redhat-5.7-x86_64.eb [hunk 2]
```diff
@@ -11,7 +11,6 @@ source_urls = ['http://content.allinea.com/downloads/']
 sources = ['%(namelower)s-tools-%(version)s.tar']
 
 # license file
-import os
-license_file = os.path.join(os.getenv('HOME'), "licenses", "allinea", "license.lic")
+license_file = HOME + 'licenses/allinea/license.lic'
```
Comment (boegel) [inline easybuild/easyconfigs/a/Allinea/Allinea-4.1-32834-Redhat-5.7-x86_64.eb]: @akesandgren there's a `/` missing before `licenses`, my bad, should be:

```python
license_file = HOME + '/licenses/allinea/license.lic'
```

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2414.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.12
See https://gist.github.com/4a06ff1ee22bb971b4556cdd095daefc for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2072.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/1f4df8cb66a1a79ef7d325a7b142109a for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**